packages/node_modules/@webex/internal-plugin-calendar/src/index.js

```diff
@@ -65,22 +65,40 @@ registerInternalPlugin('calendar', Calendar, {
             return Promise.resolve();
           }
 
-          // Decrypt participant properties
-          const decryptedParticipants = object.encryptedParticipants.map((participant) => Promise.all([
-            ctx.transform('decryptTextProp', 'encryptedEmailAddress', object.encryptionKeyUrl, participant),
-            ctx.transform('decryptTextProp', 'encryptedName', object.encryptionKeyUrl, participant)
-          ]));
+          // Check if notes and participants were sent with meeting object, and fetch if not
```

Comment:
These changes may require modification/refactor of the tests to make sure it works as expected and does not fail during CI process. Please make changes to the testing files.